core/src/ru/spbau/blackout/utils/InplaceSerializable.java

```diff
@@ -0,0 +1,20 @@
+package ru.spbau.blackout.utils;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+public interface InplaceSerializable {
```

Comment:
I'm sorry, I had to write comments about that. We tried a lot of approaches, but this is only one which seems to be able to do what we want.
The problem is that we have many fields which aren't serializable and many things which we don't want to send via network for efficiency purpose.
For instance, `GameObject` has a non-serializable field `body`, but we want to send transform (position and rotation) which is a part of body. It's impossible to override desserialization correctly because we need initialized `Body` object to assign transform in. That's why we have to do serialization inplace.